tools/distpartitioning/data_shuffle.py

```diff
@@ -92,37 +94,42 @@ def gen_node_data(rank, world_size, id_lookup, ntid_ntype_map, schema_map):
         using information present in the metadata json file
 
     '''
-    local_node_data = { constants.GLOBAL_NID : [],
-                        constants.NTYPE_ID : [],
-                        constants.GLOBAL_TYPE_NID : []
-                        }
-
+    local_node_data = {}
+    for local_part_id in range(num_parts//world_size):
```

Comment:
This was intentional. I will run pylint, flake and other tools in a separate PR. This will keep the amount of code changes in this PR to be very small. Otherwise if we run these tools and this will result in additional code changes which are unrelated to the functionality of this PR. I have already mentioned this to Steve about running these tools in a separate PR.